includes/html/pages/device/loadbalancer/alteonos_common.inc.php

```diff
@@ -0,0 +1,307 @@
+<?php
+
+use App\Models\Sensor;
+
+if (! function_exists('alteonos_sensor_types')) {
```

Comment:
This doesn't need to be a function.  In fact, you can remove this file and just put the arrays in the in the respective files.